main.go

```diff
@@ -711,6 +729,18 @@ func hashWith(salt, value string) string {
 	return "z" + sum[:length]
 }
 
+func encodeIntToName(i int) string {
+	const privateNameCharset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_"
```

Comment:
You could make this more efficient by using `unicode.Letter` or even `unicode.Letter` and `unicode.Digit` 

from the spec `identifier = letter { letter | unicode_digit }`

https://golang.org/ref/spec#identifier
https://golang.org/ref/spec#letter